app/views/pages/home.html.erb

```diff
@@ -1,2 +1,4 @@
-<h1>Pages#home</h1>
-<p>Find me in app/views/pages/home.html.erb</p>
+<div class="container">
+  <h1>Welcome, User!</h1>
```

Comment:
I think this could be the actual user name if they are logged in. You could use `current_user.present?`